### ISSUE TITLE
The way of accuracy checking in multi line test is not correct.

### ISSUE THREAD

**Issue by yumere** [mjc92/JavaScriptExtract#8]
When test, accuracy checking proceed as shown in code below.

```python
### in main.py
packed_outputs, packed_targets = pack_padded(outputs, targets)
predicted = packed_outputs.max(1)[1]
correct = (predicted == packed_targets).long().sum().data[0]

### pack_padded(outputs, targets) in packages/functions.py
out_pack = []
    tar_pack = []
    target_lens = (targets>0).long().sum(1).data
    for i,length in enumerate(target_lens):
        out_pack.append(outputs[i,:length])
        tar_pack.append(targets[i,:length])
    return torch.cat(out_pack,0), torch.cat(tar_pack,0)
```

According the above code, `<UNK>`, `<EOS>` are included in **predicted outputs** and **targets**.

As I think, `<UNK>` and `<EOS>` should not be included in correct words.

So I tested with same test dataset and model without `<UNK>`, `<EOS>`.

I got results as
similarity accuracy: **64.7%**
total accuracy: **62.3%**

But in case of testing with `<UNK>`, `<EOS>` tokens, I got results as
similarity accuracy: **63.6%**
total accuracy: **66.9%**

In conclusion, when include `<UNK>`, `<EOS>` tokens I've got accuracy **4.6% better**.
